docs/network-iot/devices/development/opensourceinitiative/paxcounter.mdx

```diff
@@ -19,7 +19,7 @@ Paxcounter is a ESP32 based device to meter pax flows in realtime
 
 ### Resources
 
-- [Product Page](https://github.com/cyberman54/ESP32-Paxcounter)
+- [Product Page](https://de.aliexpress.com/item/32915894264.html)
```

Comment:
Is there a version of this link that isn't specific to German?

Reply:
Thanks for reviewing. I changed it.